Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml

```diff
@@ -562,7 +562,11 @@ tasks:
       val2:
         simple: n/a
       val3:
-        simple: n/a
+        complex:
+          root: AWS.EC2.Instances.NetworkInterfaces.Association
+          accessor: IpOwnerId
+          transformers:
+          - operator: FirstArrayElement
```

Comment:
@johnnywilkes is this always a list of 1?

Reply:
its is always a list of at least one and they should always be the same because I don't think you can have interfaces that are in different accounts.  It seemed safe to just pick the first one to make sure it was a single string instead of list.